sycl/test-e2e/ESIMD/local_accessor_gather_scatter.cpp

```diff
@@ -83,6 +80,14 @@ template <typename T, unsigned VL, unsigned STRIDE> bool test(queue q) {
     if (i != MASKED_LANE) {
       T gold = static_cast<T>(i);
 
+      if (A[i] != gold) {
+        if (++err_cnt < 35) {
```

Comment:
It is same code on L84+ and L90+.
You could avoid code-duplication by doing something like:
```c++
T gold = (i != MASKED_LANE) ? i : -1;
```